yarn-project/aztec-node/src/sentinel/sentinel.ts

```diff
@@ -174,7 +174,8 @@ export class Sentinel extends (EventEmitter as new () => WatcherEmitter) impleme
     const args = criminals.map(address => ({
       validator: EthAddress.fromString(address),
       amount: this.config.slashInactivityCreatePenalty,
-      offense: Offense.INACTIVITY,
+      offenseType: OffenseType.INACTIVITY,
+      epochOrSlot: epoch,
     }));
```

Comment:
All want-to-slash events (ie offenses) now have an `epochOrSlot` identifier, so we can differentiate the same offense from the same validator at different times. Whether it's an epoch or a slot depends on the offense type.